freckle-app.cabal

```diff
@@ -1,11 +1,11 @@
 cabal-version: 1.18
 
--- This file has been generated from package.yaml by hpack version 0.35.2.
+-- This file has been generated from package.yaml by hpack version 0.35.0.
 --
 -- see: https://github.com/sol/hpack
 
 name:           freckle-app
-version:        1.12.0.0
+version:        1.13.0.0
```

Comment:
This feels aggressive. Nothing about any of the code-level interfaces have changed. As far as I can tell, this is a bugfix or _maybe_ an addition. I'd go `v1.12.0.1`, myself.

Reply:
sure, i'll do that, np. I figured adding the other metric was considered an addition, but I really _wanted_ to do this so.

Reply:
Even if it were and addition that'd only be `1.12.1.0`.

Reply:
Hmm, maybe I need to refresh on how haskell's PVP works?

Reply:
We have it documented here: https://illuminate.atlassian.net/wiki/spaces/EN/pages/17185952648/Maintaining+Haskell+Open+Source#Versioning

We do SemVer in the last 3 components and treat the 1st component as an overall "epoch". We like this because you can think in SemVer while still implicitly satisfying PVP, thus side-stepping a lot of flame wars about it.